db/schema.rb

```diff
@@ -29,7 +29,10 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.uuid "provider_id"
+    t.string "reference"
+    t.datetime "submitted_at"
```

Comment:
Since this field is added, do we need the `draft` attribute anymore? Could a record without a `submitted_at` value be considered a draft?

Reply:
I agree that the draft attribute is not suitable for our requirements anymore. I do think we should leave it like this for this PR and refactor the claim to use a status enum

Reply:
A draft claim cannot have a submitted at